### PR TITLE
ipset: Address a TODO, add test for TestEntry() with IPv6 address

### PR DESCRIPTION
**Pull request by masap**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
ipset: Address a TODO, add test for TestEntry() with IPv6 address

This TODO was introduced by commit 45ad69765e8a6c11aa704a64dffd8b97b7b4f24f "wrapper ipset util".

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
```docs
NONE
```

### CODE REVIEW

File: pkg/util/ipset/ipset_test.go [hunk 1]
```diff
@@ -502,6 +501,54 @@ func TestTestEntry(t *testing.T) {
...
+	setName := "NOT"
+	fcmd := fakeexec.FakeCmd{
+		CombinedOutputScript: []fakeexec.FakeAction{
+			// Success
+			func() ([]byte, []byte, error) {
+				return []byte("fd00:1234:5678:dead:beaf::1,tcp:8080 is in set " + setName + "."), nil, nil
+			},
+			// Failure
+			func() ([]byte, []byte, error) {
+				return []byte("fd00:1234:5678:dead:beaf::1,tcp:8080 is NOT in set FOOBAR."), nil, &fakeexec.FakeExitError{Status: 1}
```
Comment (aojea) [inline pkg/util/ipset/ipset_test.go:520]: you should change the IP here to a nonexistant one

Comment (aojea) [reply pkg/util/ipset/ipset_test.go:520]: the test doesn't seem to check it but is confusing, because it actually exists

Comment (masap) [reply pkg/util/ipset/ipset_test.go]: Thank you ! I modified it to nonexistent one.

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**